master/buildbot/status/web/hooks/github.py

```diff
@@ -100,10 +101,10 @@ def handle_push(self, payload):
         user = None
         # user = payload['pusher']['name']
         repo = payload['repository']['name']
-        repo_url = payload['repository']['url']
+        repo_url = payload['repository']['clone_url']
         # NOTE: what would be a reasonable value for project?
         # project = request.args.get('project', [''])[0]
-        project = payload['repository']['full_name']
+        project = payload['repository']['name']
```

Comment:
The current values are also quite arbitrary, if your codebases aren't using the web-facing urls instead of the actual git urls, and if project names aren't using the account/repo format...